src/firmware/boards/robot_stm32h7/io/ublox_odinw262_communicator.c

```diff
@@ -72,16 +71,16 @@ void io_ublox_odinw262_communicator_init(UART_HandleTypeDef* uart_handle,
     g_initialized = true;
 }
 
-void io_ublox_odinw262_communicator_task(void* arg)
+void io_ublox_odinw262_communicator_connectToWiFi(void)
 {
     assert(g_initialized);
 
-    io_ublox_odinw262_reset();
     io_ublox_odinw262_communicator_waitForBoot();
 
     // TODO check the response of io_ublox_odinw262_communicator_sendATCommand
     // and handle errors: https://github.com/UBC-Thunderbots/Software/issues/1875
-    io_ublox_odinw262_communicator_sendATCommand("AT+UMLA=2,00AAAAAAAA00\r");
+    io_ublox_odinw262_communicator_sendATCommand("AT\r");
+    io_ublox_odinw262_communicator_sendATCommand("AT+UMLA=2,0080E1000000\r");
```

Comment:
what does this change?

Reply:
the mac address changed, so i've changed it here as well, the ublox needs the same mac as the h7

eventually this will need to be tied to robot ID or some hardware UUID (not sure if that exists though)